application/emulators/trinity_rbac_legion_soap.php

```diff
@@ -0,0 +1,432 @@
+<?php
+
+/**
+ * Abstraction layer for supporting different emulators
+ */
+
+class Trinity_rbac_legion_soap implements Emulator
+{
+	protected $config;
+	
+	/**
+	 * Whether or not this emulator supports remote console
+	 */
+	protected $hasConsole = true;
+
+	/**
+	 * Whether or not this emulator supports character stats
+	 */
+	protected $hasStats = true;
+
+	/**
+	 * Console object
+	 */
+	protected $console;
+
+	/**
+	 * Array of expansion ids and their corresponding names
+	 */
+	protected $expansions = array(
+		6 => "Legion",
+		5 => "Warlords of Draenor",
+		4 => "Mists of Pandaria",
+		3 => "Cataclysm",
+		2 => "WotLK",
+		1 => "TBC",
+		0 => "None"
+	);
+
+	/**
+	 * Array of table names
+	 */
+	protected $tables = array(
+		"account" => "account",
+		"account_access" => "account_access",
+		"account_banned" => "account_banned",
+		"characters" => "characters",
+		"item_template" => "item_template",
+		"character_stats" => "character_stats",
+		"guild_member" => "guild_member",
+		"guild" => "guild",
+		"gm_tickets" => "gm_tickets"
```

Comment:
this is not true . Trinity 7.x.x don't have table gm_tickets , renamed to gm_bug